amundsen_application/static/js/ducks/user/types.ts

```diff
@@ -52,3 +57,42 @@ export interface GetUserResponse {
   type: GetUser.SUCCESS | GetUser.FAILURE;
   payload?: User;
 }
+
+
+/* getUserOwn */
+export enum GetUserOwn {
+  REQUEST = 'amundsen/user/own/GET_REQUEST',
+  SUCCESS = 'amundsen/user/own/GET_SUCCESS',
+  FAILURE = 'amundsen/user/own/GET_FAILURE',
+}
+
+export interface GetUserOwnRequest {
+  type: GetUserOwn.REQUEST;
+  payload: {
+    userId: string;
+  }
+}
+
+export interface GetUserOwnResponse {
+  type: GetUserOwn.SUCCESS | GetUserOwn.FAILURE;
+  payload?: Resource[];
+}
+
+/* getUserRead */
+export enum GetUserRead {
+  REQUEST = 'amundsen/user/read/GET_REQUEST',
+  SUCCESS = 'amundsen/user/read/GET_SUCCESS',
+  FAILURE = 'amundsen/user/read/GET_FAILURE',
+}
+
+export interface GetUserReadRequest {
+  type: GetUserRead.REQUEST;
+  payload: {
+    userId: string;
+  }
+}
+
+export interface GetUserReadResponse {
+  type: GetUserRead.SUCCESS | GetUserRead.FAILURE;
+  payload?: Resource[];
```

Comment:
Since we decided we wanted `payload`s to be objects for consistency, this should be:
```
payload?: { 
  someName: Resource[] ;
};
```